plugins/woocommerce/client/admin/client/wp-admin-scripts/payment-method-promotions/payment-promotion-row.tsx

```diff
@@ -151,6 +151,7 @@ export const PaymentPromotionRow: React.FC< PaymentPromotionRowProps > = ( {
 								subTitleContent ? (
 									<div
 										className="pre-install-payment-gateway__subtitle"
+										// eslint-disable-next-line react/no-danger - innerHTML from the element with class name: gateway-subtitle.
```

Comment:
I think you're missing a hyphen here? Should be `--`?

Reply:
Good catch, updated here: https://github.com/woocommerce/woocommerce/pull/54119/commits/84f51fcb67a4bef06e046b82209342e4ebf3fe53